mautrix_facebook/portal.py

```diff
@@ -2012,6 +2029,30 @@ async def handle_facebook_reaction_remove(
                 pass
             await reaction.delete()
 
+    async def handle_facebook_poll(
+        self,
+        sender: p.Puppet,
+        thread_change: mqtt.ThreadChange,
+    ) -> None:
+        if not self.mxid:
+            return
+
+        if thread_change.action_data["event_type"] != "question_creation":
+            return
+
+        question_json = json.loads(thread_change.action_data.get("question_json"))
+        options_html = "".join(f"<li>{o['text']}</li>" for o in question_json.get("options"))
+
+        await sender.intent_for(self).send_text(
```

Comment:
This needs to use `self._send_message` to be encrypted (which also requires parsing the HTML manually, `parse_html` can be found in `mautrix.util.formatter`)